backend/export.py

```diff
@@ -71,6 +119,7 @@ def fetch_tree_messages_and_avg_labels(
     prompts_only: bool = False,
     lang: Optional[str] = None,
     review_result: Optional[bool] = None,
+    limit: int = 0,
```

Comment:
Why zero instead of `Optional[int] = None` ? Since everything is none-able in python I always use None to indicate 'not set' instead of a special in-domain value (which I would use in C style code) ... but is a matter of taste of course.